content/docs/fundamentals/converting/resolving-errors@pt_BR.md

```diff
@@ -0,0 +1,350 @@
+{% endcall %}
+
+Tente abrir o arquivo externo [`base.js`] (https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.js). O que você vê? O arquivo deverá estar vazio, sem qualquer código JavaScript, e incluir apenas um comentário com informações como esta:
+
+```javascript
+/*
+
+This external JavaScript file is intentionally empty.
+
+Its purpose is merely to demonstrate the AMP validation error related to the
+use of external JavaScript files.
+
+*/
+```
+
+Considerando que esse arquivo JavaScript externo não é um componente funcional do site, é possível remover a referência inteira com segurança.
+
+**Remova** a seguinte referência externa de JavaScript do documento:
+
+```html
+<script type="text/javascript" src="base.js"></script>
+```
+
+Agora, **atualize** a página e verifique se o erro do script desapareceu.
+
+## Incluir CSS padrão da AMP
+
+Os seguintes erros se referem à ausência de código boilerplate:
+
+<pre class="error-text">
+A tag obrigatória "noscript enclosure for boilerplate" está ausente ou incorreta.
+A tag obrigatória "head > style : boilerplate" está ausente ou incorreta.
+A tag obrigatória "noscript > style : boilerplate" está ausente ou incorreta.
+</pre>
+
+Todo documento AMP exige o seguinte código boilerplate correspondente:
+
+```html
+<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+```
+
+**Adicione** o código boilerplate à parte inferior da tag `<head>` do documento.
+
+Inicialmente, a tag `<style amp-boilerplate>` oculta o conteúdo do corpo até que a biblioteca JavaScript da AMP seja carregada. Em seguida, o conteúdo é renderizado. Isso acontece nas AMP para evitar a renderização de conteúdo sem estilo, também conhecida como Flash Of Unstyled Content (FOUC). Isso ajuda a garantir que a experiência do usuário seja realmente instantânea, porque o conteúdo da página é exibido de uma só vez e tudo acima da dobra é renderizado em conjunto. A segunda tag inverterá essa lógica se o JavaScript estiver desativado no navegador.
+
+## Substituir`<img>` por `<amp-img>`
+
+A AMP não é compatível com os elementos HTML padrão correspondentes à exibição de mídia, o que explica este erro:
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).